benches/bench.rs

```diff
@@ -63,7 +63,7 @@ fn native(b: &mut Bencher) {
 
     b.iter(|| {
         for _ in 0..MAX_ITER {
-            pred_dc(&mut o2, 32, &above[..4], &left[..4]);
+        pred_dc::<Block4x4>(&mut o2, 32, &above[..4], &left[..4]);
```

Comment:
I think we just want to drop this bench since the approach it was measuring is deleted.

Reply:
You are right.

Reply:
Done